src/parser/xasl_generation.c

```diff
@@ -25111,7 +25111,7 @@ pt_to_merge_update_query (PARSER_CONTEXT * parser, PT_NODE * select_list, PT_MER
 PT_NODE *
 pt_to_merge_insert_query (PARSER_CONTEXT * parser, PT_NODE * select_list, PT_MERGE_INFO * info)
 {
-  PT_NODE *subq, *corr_subq, *expr, *and_expr, *value;
+  PT_NODE *subq, *corr_subq = NULL, *expr = NULL, *and_expr = NULL, *value = NULL;
```

Comment:
It is trivial.
```suggestion
  PT_NODE *subq = NULL, *corr_subq = NULL, *expr = NULL, *and_expr = NULL, *value = NULL;
```

Reply:
**subq** is always assigned some address as new node.